fission/src/ui/components/MainHUD.tsx

```diff
@@ -159,4 +163,29 @@ const MainHUD: React.FC = () => {
     )
 }
 
+async function TestGodMode() {
+    const robot: MirabufSceneObject = [...World.SceneRenderer.sceneObjects.entries()]
+        .filter(x => {
+            const y = x[1] instanceof MirabufSceneObject
+            return y
+        })
+        .map(x => x[1])[0] as MirabufSceneObject
+    const rootNodeId = robot.GetRootNodeId()
+    if (rootNodeId == undefined) {
+        console.error("Robot root node not found for god mode")
+        return
+    }
+    const robotPosition = World.PhysicsSystem.GetBody(rootNodeId).GetPosition()
+    const [ghostBody, _ghostConstraint] = World.PhysicsSystem.CreateGodModeBody(rootNodeId)
```

Comment:
```suggestion
    const [ghostBody, _ghostConstraint] = World.PhysicsSystem.CreateGodModeBody(rootNodeId, new JOLT.Vec3(0, 2, 0))
```
Adding the placeholder anchor point

Reply:
Shouldn't the anchor point be placed right on the node?